spec.html

```diff
@@ -530,12 +530,12 @@ <h1>Runtime Semantics: Evaluation</h1>
             OptionalExpression OptionalChain
         </emu-grammar>
         <emu-alg>
-          1. Let _baseExpression_ be the first child of this production (i.e., this |MemberExpression|, |CallExpression|, or |OptionalExpression|).
+          1. Let _baseExpression_ be the first child of this |OptionalExpression| (i.e., this |MemberExpression|, |CallExpression|, or |OptionalExpression|).
```

Comment:
I think we also need to drop this `this`, right?
```suggestion
          1. Let _baseExpression_ be the first child of this |OptionalExpression| (i.e., |MemberExpression|, |CallExpression|, or |OptionalExpression|).
```

Reply:
I think so. I will update that.